src/Content/PlainTextContentStorageHandler.php

```diff
@@ -280,6 +280,20 @@ public function read(VersionId $versionId, Language $language): array
 		return [];
 	}
 
+	/**
+	 * Searches and replaces one or multiple strings
+	 *
+	 * @throws \Kirby\Exception\NotFoundException If the version does not exist
+	 */
+	public function replaceStrings(
+		VersionId $versionId,
+		Language $language,
+		array $map
+	): void {
+		$file = $this->contentFile($versionId, $language);
+		F::replaceStrings($file, $map);
+	}
```

Comment:
One last thought here. Do we really gain so much by using the new `F::replaceStrings` method. In theory this could also mean that a field name is accidentally replaced, right? If we just keep the default behaviour from ContentStorageHandler::replaceStrings for text files, we would probably not hit huge performance issues, but at the same time we don't need the new F:: method and we also don't risk overwriting field names by accident.